k8s/pkg/apis/longhorn/v1beta2/backup.go

```diff
@@ -119,6 +119,9 @@ type BackupStatus struct {
 	// Size in bytes of reuploaded data
 	// +optional
 	ReUploadedDataSize string `json:"reUploadedDataSize"`
+	// The backup target name.
+	// +optional
+	BackupTarget string `json:"backupTarget"`
```

Comment:
nit: rename it to `BackupTargetName`

Reply:
@mantissahz could you address this comment so the field name is consistent with other CRDs?

Reply:
Modified. Thanks